sdk/dotnet/Pulumi.Automation/Pulumi.Automation.xml

```diff
@@ -1370,13 +1370,6 @@
             <param name="force">A boolean indicating whether the deletion should be forced.</param>
             <param name="cancellationToken">A cancellation token.</param>
         </member>
-        <member name="M:Pulumi.Automation.WorkspaceStackState.UnprotectAsync(System.String)">
-            <summary>
-            Unprotect a resource in a stack's state.
-            This command clears the ‘protect’ bit on the provided resource <paramref name="urn"/>, allowing the resource to be deleted.
-            </summary>
-            <param name="urn">The Pulumi URN to be unprotected.</param>
-        </member>
```

Comment:
Unrelated changes that are made automatically and should have been previously committed.